debian/changelog

```diff
@@ -1,3 +1,38 @@
+ubuntu-advantage-tools (27.14~23.04.1) lunar; urgency=medium
+
+  * d/ubuntu-advantage-tools.{postinst,postrm,preinst}:
+    - migrate certain settings out of uaclient.conf to a new file managed by
+      the pro config subcommand
+  * d/ubuntu-advantage-tools.postinst:
+    - refactor PREVIOUS_PKG_VER as a global variable
+    - simplify how we add notices
+  * api: new u.unattended_upgrades.status.v1 endpoint for querying status of
```

Comment:
I know this is a draft, but I think we should separate the debian changes from the features by using the:
`New upstream release 27.14`

Reply:
Yep my bad, I'll update it

Reply:
updated!